tests/helpers/basic.py

```diff
@@ -1,11 +1,27 @@
 #!/usr/bin/env python3
+import copy
 import random
 import unittest
+from functools import reduce
 
 import numpy as np
 import torch
 
 
+def get_nested_attr(obj, layer_name):
+    return reduce(getattr, layer_name.split("."), obj)
```

Comment:
Actually, we have exact same function here: 
https://github.com/pytorch/captum/blob/master/captum/attr/_models/base.py#L124
We should be able to reuse it.

Reply:
Yeah, I got it from there, but good point, will just access it directly, thanks!